pkg/resource/aws/aws_sqs_queue_test.go

```diff
@@ -0,0 +1,73 @@
+package aws_test
+
+import (
+	"testing"
+
+	"github.com/cloudskiff/driftctl/pkg/analyser"
+	awsresources "github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/r3labs/diff/v2"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/cloudskiff/driftctl/test/acceptance"
+	"github.com/cloudskiff/driftctl/test/acceptance/awsutils"
+)
+
+func TestAcc_AwsSqsQueue(t *testing.T) {
+	var mutatedQueue string
+	acceptance.Run(t, acceptance.AccTestCase{
+		Path: "./testdata/acc/aws_sqs_queue",
+		Args: []string{"scan", "--filter", "Type=='aws_sqs_queue'"},
+		Checks: []acceptance.AccCheck{
+			{
+				Env: map[string]string{
+					"AWS_REGION": "us-east-1",
+				},
+				Check: func(result *acceptance.ScanResult, stdout string, err error) {
+					if err != nil {
+						t.Fatal(err)
+					}
+					result.AssertInfrastructureIsInSync()
+					result.Equal(2, result.Summary().TotalManaged)
+					mutatedQueue = result.Managed()[0].TerraformId()
```

Comment:
Dammit why I haven't did this in `aws_instance` this is so cool !

Awesome test 👍🏻 👍🏻 👍🏻

Reply:
Ahah I just stole it from @moadibfr ^^

Reply:
I still have a little doubt on the ordering preservation in theses slices ...